deployments/operator/default/manager_auth_proxy_patch.yaml

```diff
@@ -19,6 +19,11 @@ spec:
         ports:
         - containerPort: 8443
           name: https
+        securityContext:
```

Comment:
off-topic but this should probably come from kubebuilder by default?

Reply:
Yeah, probably makes sense to submit a PR to their scaffolding code.